static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -942,6 +962,59 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
     );
   }
 
+  displayNoConditionsWarning(): boolean {
+    const {rule} = this.state;
+    return (
+      this.props.organization.features.includes('noisy-alert-warning') &&
+      !!rule &&
+      !isSavedAlertRule(rule) &&
+      rule.conditions.length === 0 &&
+      rule.filters.length === 0
+    );
+  }
+
+  renderAcknowledgeNoConditions(disabled: boolean) {
+    const {detailedError, acceptedNoisyAlert} = this.state;
+
+    return (
+      <Alert type="warning" showIcon>
+        <div>
+          {t(
+            'Alerts without conditions can fire too frequently. Are you sure you want to save this alert rule?'
+          )}
+        </div>
+        <AcknowledgeField
+          label={null}
+          help={null}
+          error={detailedError?.acceptedNoisyAlert?.[0]}
+          disabled={disabled}
+          required
+          stacked
+          flexibleControlStateSize
+          inline
+        >
+          <AcknowledgeLabel>
+            <Checkbox
+              size="md"
+              name="acceptedNoisyAlert"
+              checked={acceptedNoisyAlert}
+              onChange={() => {
+                this.setState({acceptedNoisyAlert: !acceptedNoisyAlert});
+                if (!acceptedNoisyAlert) {
```

Comment:
Is this saying if you haven't accepted it we log an analytics event that you did accept it?

Reply:
it'll only emit this when its setting the value to true